<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QToolBar" name="toolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionStart"/>
   <addaction name="actionDownload"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionRecover"/>
   <addaction name="actionSetting"/>
   <addaction name="ftpDown"/>
   <addaction name="ftpUp"/>
   <addaction name="actionSaveModel"/>
   <addaction name="actionLoadModel"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <action name="actionRefresh">
   <property name="icon">
    <iconset theme="刷新" resource="a.qrc">
     <normaloff>:/icons/refresh</normaloff>:/icons/refresh</iconset>
   </property>
   <property name="text">
    <string>刷新</string>
   </property>
   <property name="toolTip">
    <string>刷新</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRecover">
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/recover</normaloff>:/icons/recover</iconset>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
   <property name="toolTip">
    <string>恢复</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/download</normaloff>:/icons/download</iconset>
   </property>
   <property name="text">
    <string>下载</string>
   </property>
   <property name="toolTip">
    <string>下载</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/start</normaloff>:/icons/start</iconset>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
   <property name="toolTip">
    <string>开始</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/stop</normaloff>:/icons/stop</iconset>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
   <property name="toolTip">
    <string>停止</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/exit</normaloff>:/icons/exit</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionSetting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="a.qrc">
     <normaloff>:/icons/setting</normaloff>:/icons/setting</iconset>
   </property>
   <property name="text">
    <string>设置</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSaveModel">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icons/bookmark_folder.ico</normaloff>:/new/prefix1/icons/bookmark_folder.ico</iconset>
   </property>
   <property name="text">
    <string>存储模板</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionLoadModel">
   <property name="icon">
    <iconset>
     <normaloff>:/new/prefix1/icons/editpaste.ico</normaloff>:/new/prefix1/icons/editpaste.ico</iconset>
   </property>
   <property name="text">
    <string>使用模板</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="ftpDown">
   <property name="text">
    <string>更新</string>
   </property>
  </action>
  <action name="ftpUp">
   <property name="text">
    <string>上传</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="a.qrc"/>
 </resources>
 <connections/>
</ui>
